docs/resources/ec_deployment_traffic_filter_association.md

```diff
@@ -0,0 +1,52 @@
+---
+page_title: "Elastic Cloud: ec_deployment_traffic_filter_association"
+description: |-
+  Provides an Elastic Cloud traffic filter association resource. Allowing traffic filter rules to be associated to an Elastic Cloud deployment outside of the  control of Terraform, allowing associations to be created and deleted.
+---
+
+# Resource: ec_deployment_traffic_filter_association
+
+Provides an Elastic Cloud traffic filter association resource. Allowing traffic filter rules to be associated to an Elastic Cloud deployment outside of the  control of Terraform, allowing associations to be created and deleted.
+
+~> **Note:** `ec_deployment_traffic_filter_association` should only be used when the deployment to attach a rule to is managed outside of Terraform. To attach a rule to an `ec_deployment` managed by Terraform, use the
```

Comment:
I think it's a good idea to explain what happens if you use this resource with deployments managed with terraform, to give users a better understanding of what to expect.